Article/Article.md

```diff
@@ -152,35 +130,41 @@ Table: **Gender equality indicators, conditional on Log GDP p/c**, regressed on
 
-# 3. Discussion and conclusions
+Given the importance of these findings and their impact, and the impressive size and complexity of the entire analysis, it is of high value to conduct a detailed reconstruction of the methodology and analysis, checking the robustness of assumptions and inferences. As the first milestone, we conducted a pure replication obtaining the gender differences in economic preferences from the Gallup World Poll 2012 and Global Preference Survey dataset, using the same methodology as the authors. Unfortunately, the datasets are publicly available only in preprocessed (qualitative and quantitative items of the questionary were already mixed and standardized) and, alas, in a partially restricted form (two control variables are missing). It, to some extent, hampered the replication's comparison with the original findings. Nevertheless, the extracted gender differences were close to the ones from the original article. In addition, we ran the same analysis using robust regression instead of ordinary linear regression as the data revealed signs of non-normality and outliers: no significant changes in the distribution of gender differences in economic preferences were observed.  
 
-The original study indicates that higher economic development and higher gender equality are associated with an increase in the gender differences in preferences, and therefore rules out the social-role theory over the post-materialistic one: when more resources are available to both men and women, the stronger expression of the gender specific preferences can be seen.
+As the next step, we investigated the indexes used to estimate economic development and gender equality and their relation. First, we reveal a very strong association between Log GDP p/c and joint gender equality index (r = , P < ) and indexes approved by international organizations such as WEF, UNDP and GDI (r = , P < ; r = , P < ; r = , P < , respectively). This association was previously highlighted by other authors [REFs] but did not receive attention in the original manuscript. In the light of this strong relation and correlation between gender differences and Log GDP p/c reported by authors, the second main reported finding of strong and statistically significant association of gender differences with joint gender equality index became trivial as Log GDP p/c is a good predictor for gender equality indexes and vise versa. Only the conditional regression with control on Log GDP p/c may uncover the separate role of gender equality on gender differences. 
```

Comment:
"This association was previously highlighted by other authors [REFs] but did not receive attention in the original manuscript." --> This sentence doesn't convince me, I'd rephrase it or even delete it, since you add that "In the light of this strong relation and correlation between gender differences and Log GDP p/c reported by authors, the second main reported finding of strong and statistically significant association of gender differences with joint gender equality index became trivial"